libs/eavmlib/src/gpio.erl

```diff
@@ -76,62 +94,215 @@ start() ->
 open() ->
     open_port({spawn, "gpio"}, []).
 
+%% @depreciated Use {@link gpio:digital_read/1} instead.
```

Comment:
Is it `@deprecated`?

Reply:
Good point! You had suggested this in the [AtomVM Telegram chat](https://t.me/atomvm), but I realized we never opened an issue here to discuss that.

Reply:
I have opened issue #371 to discuss this topic.